backend/api/Areas/Project/Mapping/Search/ProjectPropertyMap.cs

```diff
@@ -0,0 +1,124 @@
+                .Map(dest => dest.PropertyTypeId,
+                    src => src.Building != null ? PropertyTypes.Building : PropertyTypes.Land)
+                .Map(dest => dest.Id, src => src.BuildingId ?? src.ParcelId)
+                .Map(dest => dest.Classification,
+                    src => src.Building != null ? src.Building.Classification.Name : src.Parcel.Classification.Name)
+                .Map(dest => dest.Name, src => src.Building != null ? src.Building.Description : src.Parcel.Description)
+                .Map(dest => dest.Address,
+                    src => src.Building != null
+                        ? src.Building.Address.FormatAddress()
+                        : src.Parcel.Address.FormatAddress())
+                .Map(dest => dest.City,
+                    src => src.Building != null ? src.Building.Address.City.Name : src.Parcel.Address.City.Name)
+                .Map(dest => dest.Municipality, src => src.Building != null ? src.Building.Parcel.Municipality : src.Parcel.Municipality)
+                .Map(dest => dest.Assessed,
+                    src => src.Building != null
+                        ? GetBuildingAssessedValue(src.Building.Evaluations)
+                        : GetParcelAssessedValue(src.Parcel.Evaluations))
+                .Map(dest => dest.NetBook,
+                    src => src.Building != null
+                        ? GetBuildingNetValue(src.Building.Fiscals)
+                        : GetParcelNetValue(src.Parcel.Fiscals))
+                .Map(dest => dest.Estimated,
+                    src => src.Building != null
+                        ? GetBuildingEstimatedValue(src.Building.Fiscals)
+                        : GetParcelEstimatedValue(src.Parcel.Fiscals))
+
+                .Map(dest => dest.AgencyCode,
+                    src => src.Building != null ? GetAgencyCode(src.Building.Agency) : GetAgencyCode(src.Parcel.Agency))
+                .Map(dest => dest.SubAgency,
+                    src => src.Building != null ? GetAgencyName(src.Building.Agency) : GetAgencyName(src.Parcel.Agency))
+
+                .Map(dest => dest.LandArea, src => src.Building != null ? src.Building.Parcel.LandArea : src.Parcel.LandArea)
+                .Inherits<BaseEntity, BaseModel>();
+
+        }
+
+        private string GetAgencyCode(Agency agency)
+        {
+            return agency.Parent?.Code ?? agency.Code;
+        }
+
+        private string GetAgencyName(Agency agency)
+        {
+            return agency != null ? agency.Name : "";
+        }
+
+        private string FormatAddress(Address address)
+        {
+            return address != null ? $"{address.Address1} {address.Address2}".Trim() : "";
+        }
+
+        private decimal GetBuildingAssessedValue(ICollection<BuildingEvaluation> evaluations)
+        {
+            return evaluations.OrderByDescending(f => f.Date).FirstOrDefault(f => f.Key == EvaluationKeys.Assessed) ==
```

Comment:
This is potentially premature optimization, but consider saving the result of the orderby so you don't have to do it twice.